### ISSUE TITLE
Reuse sheet functions

### ISSUE THREAD

**Issue by jameesjohn** [oppia/oppiabot#98]
Modify issue assigned module to use the same functions as used in apiForSheets when interacting with spreadsheets.

**Comment by anandwana001** [comment]
@jameesjohn 
The current codebase doesn't have `checkIssueAssigned ` but it does have `checkIssueAssignedSpec` file.

**Comment by jameesjohn** [comment]
Oops, my bad.
The name of the file should be `checkIssueAssignee.js`

**Comment by riyasingh1004** [comment]
Hi, I would like to work on this issue. Can someone guide me how to get started as I am still trying to understand the codebase.

**Comment by jameesjohn** [comment]
Hi, @riyasingh1004. I'm available to guide you to understand the codebase.

**Comment by riyasingh1004** [comment]
@jameesjohn thank you. Should I ask all the doubts here or is there a separate channel for the project. Also can you tell me how to get started with this issue.

**Comment by jameesjohn** [comment]
We primarily use hangouts for communication. However, make sure you have signed the CLA which can be found [here](https://goo.gl/forms/AttNH80OV0). Once you are done with that, you can send me a message on hangouts (my email is available on my profile).

Thanks.

**Comment by riyasingh1004** [comment]
Thank you, I have sent you the message request in hangout and have signed the CLA.

**Comment by gp201** [comment]
@vojtechjelinek Can I take up this issue

**Comment by nlok5923** [comment]
@vojtechjelinek if this issue is up for grabs i would like give it a try.

**Comment by MohdImran001** [comment]
Hey @jameesjohn, can I work on this issue?

**Comment by jameesjohn** [comment]
This issue is currently blocked on #195 which is why we can't assign it to anyone.

**Comment by seanlip** [comment]
Note: #195 is already done, so this issue should now be available again.

**Comment by himanshusharma27** [comment]
I want to work on this issue . Please assign me this issue .
